feat/new product overview pages (#646)
hirosystems/docs: docs/explorer/index.mdx

---
# The default id is the same as the one defined below. so not needed
title: Stacks Explorer Overview
sidebar_label: Overview
custom_edit_url: null
---

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

<p class="p-3 rounded-lg bg-neutral-100 text-neutral-500 dark:bg-neutral-800 dark:text-neutral-400">Explore and verify data on the Stacks blockchain.</p>

<div class="mb-10"></div>

The Stacks Explorer lets you explore on-chain data, troubleshoot pending transactions, edit and deploy contracts via the sandbox, discover new tokens, and more. We maintain and support the [Stacks Explorer](https://explorer.hiro.so) as a service for the community, but you can also run your own instance of the explorer.

> [View Stacks Explorer](https://explorer.hiro.so)

Reviewer: <div class="flex justify-start my-8">
  <a href="https://explorer.hiro.so" class="pr-3">
    View Stacks Explorer <span classNam="ml-1 no-underline">→</span>
  </a>
</div>

> [View Stacks Explorer](https://explorer.hiro.so)

Reviewer: Is it possible to do buttons in docusaurus? I want these CTAs to be front and center, and they get lost in the noise in this formatting.

I was picturing a two button dynamic here (one bright orange button, one silhouette button). If that's hard to execute, I'd be happy with 1 button, and we trim the secondary CTA here

[Set up your own Explorer](./getting-started.md)

### Related Tools

**[Stacks Blockchain API](../stacks-blockchain-api/index.mdx)**: Looking for on-chain data without the front end? Leverage the Stacks Blockchain API.

**[Ordinals Explorer](../ordinals-explorer/index.mdx)**: Explore Bitcoin Ordinals and BRC-20 tokens via the Ordinals Explorer.

---

> Need help building with Stacks Explorer? Reach out to us on the `#stacks-explorer` channel on Discord under Hiro Developer Tools section. We also host a weekly office hours on Discord every Monday at 11am ET.

Reviewer: ```suggestion
Need help building with Stacks Explorer? Reach out to us on the `#stacks-explorer` channel on Discord under the [Hiro Developer Tools section](https://stacks.chat/). There’s also a [weekly office hours](https://events.stacks.co/event/HD16484710) on Discord every Thursday at 11am ET.
```
